src/cgame/default/cg_main.c

```diff
@@ -124,7 +125,11 @@ static void Cg_Init(void) {
 	cg_add_weather = cgi.AddCvar("cg_add_weather", "1", CVAR_ARCHIVE,
 	                          "Control the intensity of atmospheric effects.");
 
-	cg_bob = cgi.AddCvar("cg_bob", "1", CVAR_ARCHIVE, "Controls weapon bobbing effect.");
+	cg_autoswitch = cgi.Cvar("autoswitch", "1", CVAR_USER_INFO | CVAR_ARCHIVE,
+				 "The weapon pickup autoswitch method. 0 disables, 1 autoswitches from Blaster only,"
```

Comment:
auto-switch

Reply:
auto-switch or auto_switch? I assume you meant auto_switch.